app/views/shared/_random_biertje.html.erb

```diff
@@ -4,15 +4,21 @@
       <h2 class="text-hamers-red-400 text-xl font-bold mb-4">Citaat van de pagelaad</h2>
       <div class="w-full flex justify-center">
         <div class="w-full flex space-x-3 text-sm">
-          <%= link_to @random_quote.user, class: "self-center" do %>
-            <% gravatar_for @random_quote.user, class: "h-8 w-8 rounded-full" %>
+          <% if @random_quote.anonymous? %>
```

Comment:
Misschien leuker om geen anonieme quotes als quote van de dag te pakken. 

Oplossing zou dan zijn door een anonymous en non_anonymous scope in de model te defineren. Dan kun je in deze controller Quote.non_anonymous gebruiken ipv Quote.